go/libkb/osv_windows.go

```diff
@@ -0,0 +1,12 @@
+// +build windows
+
+package libkb
+
+// OSVersionAndBuild returns OS version, and build too on some platforms
+func OSVersionAndBuild() (string, string, error) {
+	productVersion, err := execToString("cmd", []string{"/c", "ver"})
```

Comment:
I thought @heronhaye worked on something like this before, maybe it could be used?

Reply:
Looks like we have it in the status package and in the install package. Could both depend on a libkb helper but doesn't really matter.

Reply:
looks like it is used, sorry! but factored out to libkb?

Reply:
Ah didn't see rest of PR. Looks fine to me

Reply:
In order to call it from libkb it had to move either into libkb or another package. Due to libkb->status->libkb